spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -613,6 +615,7 @@ public void testDeployWithProperties() throws Exception {
 		assertEquals("2", logAppProps.get("spring.cloud.stream.instanceCount"));
 		assertEquals("true", logAppProps.get("spring.cloud.stream.bindings.input.consumer.partitioned"));
 		assertEquals("3", logAppProps.get("spring.cloud.stream.bindings.input.consumer.concurrency"));
+		assertEquals("2", logAppProps.get(StreamPropertyKeys.INSTANCE_COUNT));
```

Comment:
maybe we should add this same assertion in other relevant places in this test class